src/protocol/commands/_vFile_pread.rs

```diff
@@ -0,0 +1,28 @@
+use super::prelude::*;
+
+#[derive(Debug)]
+pub struct vFilePread {
+    pub fd: usize,
+    pub count: usize,
+    pub offset: usize,
```

Comment:
For correctness sake, these ought to be parsed into `&'a [u8]` buffers, and then converted into `Target::Arch::Usize` somewhere in the `vFile` handler code.

Using `usize` here would mean, for example, that an emulator running on a 32-bit platform would not be able to work with 64-bit guests.

For an example of how this conversion is done elsewhere in `gdbstub`, check out:

https://github.com/daniel5151/gdbstub/blob/dev/0.6/src/gdbstub_impl/ext/breakpoints.rs#L19-L20